docs/data/material/getting-started/templates/landing-page/LandingPage.js

```diff
@@ -52,7 +52,9 @@ function ToggleCustomTheme({ showCustomTheme, toggleCustomTheme }) {
           Custom theme
         </ToggleButton>
         <ToggleButton value={false}>
-          <SvgMaterialDesign sx={{ fontSize: '20px', mr: 1 }} />
+          <Typography fontWeight={800} sx={{ mr: 1 }}>
```

Comment:
Do we need this? We can just say "Material Design 2" and remove this additional element.